pastas/stats/signatures.py

```diff
@@ -164,27 +287,30 @@ def _martens(series: Series, normalize: bool = True) -> Tuple[Series, Series]:
     Returns
     -------
     hl: pandas.Series
-        Lowest heads
+        Lowest heads in a year.
```

Comment:
Is this correct? When I look at the code it seems more like the mean of the 3 lowest monthly heads